### PR TITLE
Add test suite

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Add tests using Jest.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -6,5 +6,8 @@
     "rootDir": "src",
     "alwaysStrict": true,
     "esModuleInterop": true
-  }
+  },
+  "exclude": [
+    "test"
+  ]
```
Comment (thewilkybarkid) [inline tsconfig.json]: Surprised I have to include this. Maybe `rootDir` doesn't do what I think it does?

**Review by giorgiosironi — APPROVED** [review]
Infrastructurally, all makes sense

File: tsconfig.json (hunk 1 above)
Comment (stephenwf) [reply tsconfig.json]: You may need to re-add `node_modules` here, I'm not entirely sure. I think `rootDir` is just where your source that will be emitted is, the project will be relative to the `tsconfig.json`. `tsc` will still try to type-check things not in `rootDir`.
> The "exclude" property defaults to excluding the node_modules, bower_components, jspm_packages and <outDir> directories when not specified.

Comment (thewilkybarkid) [reply tsconfig.json]: https://github.com/microsoft/TypeScript/issues/9858 is relevant.

Comment (thewilkybarkid) [reply tsconfig.json]: I read that as use `"include": ["src"]` instead.

Comment (thewilkybarkid) [reply tsconfig.json]: https://github.com/libero/article-store/pull/8

**Pull request merged**